docs/getting-started/retrieving-keys.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+---
+
+# Key Retrieval Guide
```

Comment:
nit: maybe "Key Download Guide"  Retrieval somehow implies reobtaining.

And to Seth's point below, we could distinguish Temporary Exposure Key vs export signing key vs apk signing key throughout.

Reply:
changed retrieve to download in a few places I had worded this different in the edit for Seth.

Please let me know if any 'key' needs clarification yet.